doc/api/stream.md

```diff
@@ -570,24 +570,24 @@ Readable stream implementation.
 Specifically, at any given point in time, every Readable is in one of three
 possible states:
 
-* `readable._readableState.flowing = null`
-* `readable._readableState.flowing = false`
-* `readable._readableState.flowing = true`
+* `readable.flowing = null`
```

Comment:
Perhaps this part should be rewritten as it appears to give the illusion that you can assign values via this property.